app/models/concerns/foreman_ansible/hostgroup_extensions.rb

```diff
@@ -36,3 +36,15 @@ def all_ansible_roles
     end
   end
 end
+
+class Hostgroup
+  apipie :class do
+    property :all_ansible_roles, array_of: 'AnsibleRole', desc: 'Returns all ansible roles assigned to the host group, both its own and inherited from parent host groups'
+    property :ansible_roles, array_of: 'AnsibleRole', desc: 'Returns ansible roles directly assigned to the host group'
+    property :inherited_ansible_roles, array_of: 'AnsibleRole', desc: 'Returns only the inherited ansible roles assigned to the host group parents'
```

Comment:
```suggestion
    property :inherited_ansible_roles, array_of: 'AnsibleRole', desc: 'Returns only the inherited ansible roles assigned to Hostgroup\'s parents'
```

Reply:
I actually checked the form we use in apipie documentation and for consistency, I kept it lowcase and two words form (in both docs). Just added `'s` as suggested.